### DISCUSSION TITLE
Mix and Match?

### DISCUSSION THREAD

**Discussion by Leonardo-Ferreira** [kedacore/http-add-on#938] [Q&A]
How can I mix and match scalers? I would like to use the Http-Add-On to scale the app TO ZERO during inactivity periods, but once "awake" I would like to scale based on other metrics, like avg response time... is it possible?

**Comment by thomasfroehle** [comment]
This is described here: https://github.com/kedacore/http-add-on/blob/main/docs/walkthrough.md#integrating-http-add-on-scaler-with-other-keda-scalers

**Reply by Leonardo-Ferreira** [reply]
I don't know but I feel like im looking for the opposite. Ideally, the OTHER scaledObject (like scale based on prometheus metric) would be the one tagged with "skip-scaledobject-creation", because it would scale from 0 to N, agree?
